apps/client/src/app/components/login-with-access-token-dialog/login-with-access-token-dialog.html

```diff
@@ -8,12 +8,12 @@
   <div class="align-items-center d-flex flex-column">
     <mat-form-field appearance="outline" class="w-100">
       <mat-label i18n>Security Token</mat-label>
-      <textarea
```

Comment:
Can you remove the style in `login-with-access-token-dialog.scss`:

```
textarea.mat-input-element.cdk-textarea-autosize {
  box-sizing: content-box;
}
```

Reply:
I removed that.